### ISSUE TITLE
Regression in wrapping of exception message

### ISSUE THREAD

**Issue by GawainLynch** [bolt/bolt#5777]
![image](https://cloud.githubusercontent.com/assets/1427081/18579449/fb874092-7bf5-11e6-93c6-46c83affd82a.png)

In the above example, I can't even get the end of the exception and it is therefore a 100% useless page, unless we add a :beer: tap of course :wink:

**Comment by GawainLynch** [comment]
The `<code>` block needs CSS equivalent to (pseudo):

```
.exception code {
    display:block; 
    white-space: pre-wrap;
}
```

**Comment by bobdenotter** [comment]
Fixed by #5790
